xhu.py

```diff
@@ -34,6 +38,14 @@
 app.config.from_envvar("XMPP_HTTP_UPLOAD_CONFIG")
 application = app
 
+if app.config['LOGDIR']:
+    logging.basicConfig(
+        filename='{}/xhu.log'.format(app.config['LOGDIR']),
+        filemode='a',
+        level=logging.INFO
+    )
+logger = logging.getLogger(__name__)
```

Comment:
I prefer to log to standard output and have the service manager deal with it. Alternatively, support systemd journal (optionally) directly. Configuring and managing log directories is a PITA, and always opens up the question of logrotate.

Reply:
> I prefer to log to standard output and have the service manager deal with it.

What service manager are you talking about here? I use Circus to run the processes, but I don't see my logs ending up in its log, so I'm not sure what you mean.

Reply:
Stuff like systemd. Even sending to syslog is better than having to work with logfiles.